### PR TITLE
feat: update pool detail for beets

### PR DESCRIPTION
**Pull request by groninge01**

### CODE REVIEW

**Comment by groninge01** [comment]
reverted moving the veballockprovider as that was causing errors on the detail (and portfolio) page

**Review by agualis — APPROVED** [review]

File: packages/lib/modules/pool/actions/add-liquidity/modal/StakingOptions.tsx [hunk 1]
```diff
@@ -54,32 +62,34 @@ export function StakingOptions() {
...
-                  isOpen={auraDisclosure.isOpen}
-                  onClose={auraDisclosure.onClose}
-                  partner={RedirectPartner.Aura}
-                  redirectUrl={getAuraPoolLink(getChainId(chain), pool.staking.aura.auraPoolId)}
-                />
-              </>
-            )}
-          </VStack>
-        </Card>
+        {isBalancerProject() && pool.staking?.aura && (
```
Comment (garethfuller) [inline packages/lib/modules/pool/actions/add-liquidity/modal/StakingOptions.tsx:65]: I'm a bit worried about conditional rendering like this within the lib folder. I guess somehow we need to split components like this out into sub-project-specific packages. But I realise you're in a rush so we can do this for now.

File: packages/lib/config/getProjectConfig.ts [hunk 2]
```diff
@@ -14,3 +14,11 @@ const PROJECT_CONFIG = process.env.NEXT_PUBLIC_PROJECT_ID
...
   return PROJECT_CONFIG
 }
+
+export function isBeetsProject() {
+  return getProjectConfig().projectId === 'beets'
+}
+
+export function isBalancerProject() {
+  return getProjectConfig().projectId === 'balancer'
+}
```
Comment (garethfuller) [inline packages/lib/config/getProjectConfig.ts:24]: Can these not just be consts? 
```suggestion
export const isBeetsProject = getProjectConfig().projectId === 'beets'
export const isBalancerProject = getProjectConfig().projectId === 'balancer'
```

File: packages/lib/modules/pool/actions/add-liquidity/modal/StakingOptions.tsx [hunk 3]
```diff
@@ -24,13 +25,15 @@ export function StakingOptions() {
 
   const auraDisclosure = useDisclosure()
 
+  const projectConfig = getProjectConfig()
```
Comment (garethfuller) [inline packages/lib/modules/pool/actions/add-liquidity/modal/StakingOptions.tsx:28]: Same with this, can this not just be a global const?

**Review by garethfuller — APPROVED** [review]
Approved to unblock, but please address comments where relevant.

File: packages/lib/config/getProjectConfig.ts (hunk 2 above)
Comment (groninge01) [reply packages/lib/config/getProjectConfig.ts]: yeah when you also implement the comment below -> updated

File: packages/lib/modules/pool/actions/add-liquidity/modal/StakingOptions.tsx (hunk 3 above)
Comment (groninge01) [reply packages/lib/modules/pool/actions/add-liquidity/modal/StakingOptions.tsx]: added but i haven't changed it everywhere yet -> #325

**Pull request merged**